internal/command/deploy/machines.go

```diff
@@ -563,25 +563,25 @@ func (md *machineDeployment) resolveUpdatedMachineConfig(origMachineRaw *api.Mac
 	}
 
 	launchInput.Config.Image = md.img.Tag
-	launchInput.Config.Metrics = md.appConfig.Metrics
 	launchInput.Config.Restart = origMachineRaw.Config.Restart
-	for _, s := range md.appConfig.Statics {
-		launchInput.Config.Statics = append(launchInput.Config.Statics, &api.Static{
-			GuestPath: s.GuestPath,
-			UrlPrefix: s.UrlPrefix,
-		})
-	}
-	launchInput.Config.Env = make(map[string]string)
-	for k, v := range md.appConfig.Env {
-		launchInput.Config.Env[k] = v
-	}
+	launchInput.Config.Guest = origMachineRaw.Config.Guest
```

Comment:
you lost the nil check here https://github.com/superfly/flyctl/pull/1810/files#diff-4251d8215f73cf413b3febd989c9853c5c6d80d94c5b8ced61697ff05c8e563fL596 unclear if its necessary but its maybe its a chesterson's fence thing.

Reply:
you are right, my line of thoughts is that `launchInput.Config.Guest` is nil already so overriding it by `origMachineRaw.Config.Guest` in case the later is nil is the same outcome.